src/towncrier/test/test_builder.py

```diff
@@ -81,3 +81,9 @@ def test_strip_with_counter(self):
             parse_newfragment_basename("  007.feature.3", ["feature"]),
             ("7", "feature", 3),
         )
+
+    def test_unlinked_feature(self):
```

Comment:
a docstring describing why this tests exist , and what is the input and expect result would be great.

The docstring is the only way we can test that the test is ok :)

Without a docstring, a test can do anything and still be a "good" test :)

Reply:
parse_newfragment_basename doesn't do the orphan magic now, so this test was removed.